society-for-american-archaeology.csl

```diff
@@ -103,20 +91,31 @@
       <substitute>
         <names variable="editor"/>
         <names variable="translator"/>
+        <names variable="director"/>
       </substitute>
     </names>
-    <text macro="anon"/>
-    <text macro="recipient"/>
+    <text macro="title"/>
+    <choose>
+      <if type="map" match="any">
+        <text variable="publisher" prefix=" (" suffix=")"/>
+      </if>
+    </choose>
   </macro>
   <macro name="contributors-short">
     <names variable="author">
       <name form="short" and="text" delimiter=", "/>
+      <label form="short" prefix=", "/>
```

Comment:
sorry, one more question -- I don't see this in the style guide and adding , eds. to in-text citations for author-date styles, while not unheard of, is very rare. Can you double check?